packages/generate/src/queries.ts

```diff
@@ -4,6 +4,12 @@ import { type CommandOptions } from "./commandutil";
 import { headerComment } from "./genutil";
 import { type Target, camelify } from "./genutil";
 
+function collectAndPrintError(errs:string[], path:string, err:any){
+  const message = `Error in file '${path}': ${err.toString()}`
+  errs.push(message)
+  console.log(`\x1b[31m${message}\x1b[0m`);
+}
```

Comment:
Instead of mutating the input array, I'd prefer this function just formats and returns a string, and leave it up to the caller to figure out what to do with the string.

Something like:

```suggestion
function formatError(path: string, err: any){
  const message = `Error in file '${path}': ${err.toString()}`
  return `\x1b[31m${message}\x1b[0m`;
}
```